src/scss/main.scss

```diff
@@ -53,11 +57,11 @@ form input{
   border:3px solid #efefef;
 }
 
-a#schedule, a#confirm-button {
+a#schedule {
   border:1px solid ;
   background-color:#ff5817;
   height:80px;
-  width:345.56px;
+  width:393.83px;
```

Comment:
It's usually a red flag when a width is hardcoded in pixels. I think it would be best to make this button use bourbon to control it's width. Something like 4 columns with 8 columns of offset (whatever the input above it is)